Wikipedia/Code/Article as a Living Document/ArticleAsLivingDocController.swift

```diff
@@ -69,24 +77,21 @@ class ArticleAsLivingDocController: NSObject {
     var shouldAttemptToShowArticleAsLivingDoc: Bool {
         
         guard let delegate = delegate,
-              let view = delegate.view else {
+              let view = delegate.view,
+              view.effectiveUserInterfaceLayoutDirection == .leftToRight,
+              delegate.articleURL.host == Configuration.Domain.englishWikipedia else {
```

Comment:
No change needed, just wanted to provide more detail about my previous rationale for having `delegate.articleURL.host == Configuration.Domain.englishWikipedia` higher up in the `guard` if it's helpful at all to read (or if it seems I'm thinking about it incorrectly, lemme know 🥴):

Typically, I always treat the order of these guard statements as most restrictive (general) to least restrictive (specific) and least expensive performance wise to most. So in the case of these `.leftToRight` and `englishWikipedia`, to me they are both probably effectively equivalent performance wise. But restriction wise, the `englishWikipedia` restricts the audience we're interested in here much more widely than the `.leftToRight` check.

It's probably some sense of silly premature optimization and organization on my part, but I think sometimes that type of precedence organization also has benefits beyond performance gains, like readability.

Reply:
Makes sense, I can see how that helps with readability. I pushed a quick commit shifting this and hopefully fixing your reverse priorities of the prompts if the user is not in the A/B test. I'm confirming there were not regressions related to that part now though, if I find any I'll revert. Good catch on that!

Reply:
I've gone through the test cases again with my last change - they still look good and the survey shows up before fundraising now if they don't land in the experiment. I do notice the fundraising announcement occasionally doesn't show up until an additional launch or explore feed refresh, which is a bit unexpected but actually more in line with the ideal functionality, that is that they don't see back to back modals. Since Josh sounded okay with back to back modals anyways, I figure this slight inconsistency is okay. I'll mention in sync tomorrow.